inc/layers.php

```diff
@@ -496,13 +496,9 @@ function get_layer($post_id = false) {
         $type = $this->get_layer_type();
 
         //$content = apply_filters('the_content', $post->post_content);
-        if (function_exists('qtrans_use')){
-            $content = qtrans_use(odm_language_manager()->get_current_language(), $post->post_content,false);//get content by langauge
-            $excerpt = qtrans_use(odm_language_manager()->get_current_language(), $post->excerpt,false);//get content by langauge
-        }else {
-            $content =  $post->post_content;//get content by langauge
-            $excerpt =  $post->excerpt;//get content by langauge
-        }
+        $content = apply_filters('translate_text', $post->post_content, odm_language_manager()->get_current_language());
```

Comment:
@Huyeng @S-mardii @DragonVirus According to https://qtranslatexteam.wordpress.com/faq/#DevelperNotes, we should not use the qTrans* functions directly. They recommend the apply_filters function. Please see the rest of this PR and keep in mind this way of pulling translated strings from qTranslateX. Also, I recommend you to read through the docs for developers which are very useful